src/doc/book/testing.md

```diff
@@ -586,3 +586,46 @@ you add more examples.
 
 We haven’t covered all of the details with writing documentation tests. For more,
 please see the [Documentation chapter](documentation.html).
+
+# Testing and concurrency
+
+One thing that is important to note when writing tests are run concurrently
+using threads (by default the number of threads is equal to the number of CPUs
+on the machine). For this reason you should take care that your tests are
```

Comment:
Could you remove this? It's not exactly true ish, and it's not something we guarantee.